lib/index.js

```diff
@@ -27,3 +30,17 @@ color.furthest = function(target, relative, bc) {
 
     return result[key];
 };
+
+color.closest_lab = function(target, relative) {
+    var key = color.lab_palette_map_key(target);
+    var result = color.map_palette_lab([target], relative, 'closest');
+
+    return result[key];
+};
+
+color.furthest_lab = function(target, relative) {
+    var key = color.lab_palette_map_key(target);
+    var result = color.map_palette_lab([target], relative, 'furthest');
```

Comment:
Why not use match directly here? From your previous comment it should yield slightly better performance. Same for closest_lab.